webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -327,6 +328,7 @@ var CURRENT_GROUP_DATA={};
         var MCK_MODE = appOptions.mode;
         MCK_LABELS = appOptions.labels;
         MCK_BASE_URL = appOptions.baseUrl;
+        var MCK_BOT_API = appOptions.botPlatformAPI;
```

Comment:
@hardikparashar Can customers update botPlatformAPI value? 
If not then don't put it in appOptions.